client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -242,13 +252,17 @@ const Checkout = React.createClass( {
 			} );
 		}
 
+		if ( ! selectedSiteSlug ) {
```

Comment:
I've changed the routes in my most recent commit, without a `selectedSiteSlug` they now default to:
plans: `/checkout/thank-you/plans`
features: `/checkout/thank-you/features`

Both show the same view but it seems good to differentiate :)